src/transac/components/alerts/alerts.service.coffee

```diff
@@ -0,0 +1,8 @@
+angular.module('transac.alerts').service('TransacAlertsService', (toastr, TOASTR_OPTS)->
```

Comment:
Sure, when you say "more talk" does that mean you think I should rename it to notifications ?